plugins/module_utils/_api_data.py

```diff
@@ -708,7 +708,10 @@ def join_path(path):
             fully_understood=True,
             primary_keys=('name', ),
             fields={
+                'comment': KeyInfo(),
+                'copy-from': KeyInfo(),
```

Comment:
Isn't `copy-from` a parameter for creation only, and not a propery of the actual pools?

Reply:
Yes good point. Do I mark it for creation only somehow or should I remove it?

Reply:
I would remove it. `copy-from` is a parameter to make creation of new entries easier from the command line.